src/python/gudhi/sklearn/__init__.py

```diff
@@ -0,0 +1,3 @@
+from .cover_complex import *
+
+__all__ = ["NerveComplex", "GraphInducedComplex", "MapperComplex"]
```

Comment:
Nerve_GIC depends on CGAL (through the bottleneck distance). When CGAL is not present, this automatic import also breaks all the other content of the sklearn submodule (currently the cubical complex persistence).